include/libtorrent/aux_/session_impl.hpp

```diff
@@ -222,7 +222,7 @@ namespace libtorrent
 			struct session_plugin_wrapper : plugin
 			{
 				explicit session_plugin_wrapper(ext_function_t const& f) : m_f(f) {}
-				explicit session_plugin_wrapper(session_plugin_wrapper const& p) : m_f(p.m_f) {}
+				explicit session_plugin_wrapper(session_plugin_wrapper const& p) = default;
```

Comment:
it fails with clang, reverting it
```
../include/libtorrent/extensions.hpp:188:11: error: definition of implicit copy constructor for 'plugin' is deprecated because it has a user-declared destructor [-Werror,-Wdeprecated]
  virtual ~plugin() {}
          ^
../src/session.cpp:335:12: note: in instantiation of function template specialization 'boost::make_shared<libtorrent::aux::session_impl::session_plugin_wrapper, libtorrent::aux::session_impl::session_plugin_wrapper>' requested here
    boost::make_shared<wrapper>(wrapper(create_ut_pex_plugin)),
           ^
../include/libtorrent/aux_/session_impl.hpp:225:14: note: implicit copy constructor for 'plugin' first required here
    explicit session_plugin_wrapper(session_plugin_wrapper const& p) = default;
             ^
/usr/local/include/boost/smart_ptr/make_shared_object.hpp:254:17: note: implicit copy constructor for 'libtorrent::aux::session_impl::session_plugin_wrapper' first required here
    ::new( pv ) T( boost::detail::sp_forward<Args>( args )... );
                ^
1 error generated.
```